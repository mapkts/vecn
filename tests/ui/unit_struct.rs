#[vecn::vector]
struct Unit;

fn main() {}
