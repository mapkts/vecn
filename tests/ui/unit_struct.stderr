error: unit struct cannot be made into a vector
 --> $DIR/unit_struct.rs:2:1
  |
2 | struct Unit;
  | ^^^^^^^^^^^^
